tests/lib/linter/linter.js

```diff
@@ -1826,7 +1826,7 @@ describe("Linter", () => {
             linter.defineRule(code, {
                 create: context => ({
                     Literal(node) {
-                        context.report(node, context.getFilename());
+                        context.report(node, context.filename);
```

Comment:
I think we should keep testing `context.getFilename()` too.

Perhaps we could add an `assert.strictEqual(context.getFilename(), context.filename);` line everywhere we now use `context.filename` in the `linter` tests? For example, here:

```suggestion
                        assert.strictEqual(context.getFilename(), context.filename);
                        context.report(node, context.filename);
```

Reply:
@mdjermanovic Done, thanks for the suggestion.